zulipterminal/ui_tools/views.py

```diff
@@ -367,7 +367,7 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
         if is_command_key("SEARCH_STREAMS", key):
             _, self.focus_index_before_search = self.log.get_focus()
             self.set_focus("header")
-            self.stream_search_box.set_caption("")
+            self.stream_search_box.set_caption(" ")
```

Comment:
What do you think about adding an extra " " in a method that overrides urwid's `set_caption` method in `PanelSearchBox` instead?

Reply:
I was instead thinking, during the Panel refactor I could group this and 1 other common line into a function call `enter_search_mode` inside PanelSearchBox

Reply:
This would be a good function to add during the first commit - it avoids spreading general `set_caption` calls through the code, which may also be used for other things, as well as providing a central place for this kind of change - and it's easy to grep for a unique name :)

Reply:
@preetmishra @neiljp Previously, the other common line that I was talking about was `enter_editor_mode` but now that I see that enter_editor_mode would be removed after the keypress refactor and set_caption would be called only once anyway after the Panel refactor, I don't think this needs to be moved into PanelSearchBox using enter_search_mode or similar.